.github/workflows/continuous-integration-workflow.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           - '2.3.6'
           - '2.4'
           - '2.5'
+          - '2.5.6'
```

Comment:
Please add the skipped versions as comments (see 1.10 above)

Reply:
Sure. https://github.com/disney/meteor-base/pull/100/commits/8c0306f2ed5152a448609e854ee5482481b2bd88